fission/function.go

```diff
@@ -71,6 +71,11 @@ func printPodLogs(c *cli.Context) error {
 
 func getInvokeStrategy(minScale int, maxScale int, executorType string, targetcpu int) fission.InvokeStrategy {
 
+	// set default minScale to 1
+	if minScale == 0 {
```

Comment:
With this change, how will a user specify minScale = 0?  If you want to default to zero you have to do it where you have the cli object, then you can check `c.IsSet("minscale")` or something like that to see if the user specified a minscale